src/boundingboxes.jl

```diff
@@ -88,49 +88,20 @@ _bboxes(boxes) = _pboxes(point for box in boxes for point in extrema(box))
 
 _pboxes(points) = _pboxes(manifold(first(points)), points)
 
-function _pboxes(::Type{𝔼{1}}, points)
+function _pboxes(::Type{𝔼{N}}, points) where {N}
   p = first(points)
   ℒ = lentype(p)
-  xmin = typemax(ℒ)
-  xmax = typemin(ℒ)
-  for p in points
-    c = convert(Cartesian, coords(p))
-    xmin = min(c.x, xmin)
-    xmax = max(c.x, xmax)
-  end
-  Box(withcrs(p, (xmin,)), withcrs(p, (xmax,)))
-end
+  cmin = fill(typemax(ℒ), N)
+  cmax = fill(typemin(ℒ), N)
```

Comment:
You can use Generated Functions to avoid the type instability caused by using `Vector` in this line.
https://docs.julialang.org/en/v1/manual/metaprogramming/#Generated-functions

Reply:
Hi @rmcaixeta, I was very busy today. Tomorrow I will help you by writing this function.

Reply:
Done!